src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,7 +27,7 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
-            + PREFIX_MATRIC_NUMBER + "MATRICULATION NUMBER "
+            + "[" + PREFIX_MATRIC_NUMBER + "MATRICULATION NUMBER] "
```

Comment:
good catch to add square brackets around the `MATRICULATION NUMBER` parameter to show users that it is an optional parameter :)